### ISSUE TITLE
tini docker windows container

### ISSUE THREAD

**Issue by maldag** [krallin/tini#216]
Is tini also used in docker windows container? Do you offer a release to use when working with docker windows containers?

**Comment by krallin** [comment]
No, the problems solved and solutions used by Tini are only relevant on UNIX
